source/app-sharing.rst

```diff
@@ -16,61 +18,67 @@ Overview
    home directory that other users can launch. This is similar to adding to your
    PATH the bin directory of another user.
 
-The rest of this document addresses OnDemand's support for Executable Sharing.
+File System Permissions
+-----------------------
```

Comment:
At the end of this section, a short paragraph with an example / demo, and then a short paragraph of why this works (changing file permissions on app directories).

For example, you could start by chmod-ing an app to 750 and show the result in the dropdown menu: the app doesn't appear (screenshot 1). Then chgrp of the directory to the group you are in. Reload the dashboard and the link now appears.

The point is that we don't call out here the fact that you can just change the permissions and reload the dashboard in your browser to see the change.